src/js/edu-benefits/5490/config/form.js

```diff
@@ -241,7 +239,7 @@ const formConfig = {
                 }
               }),
               'ui:options': {
-                hideIf: (fieldData) => fieldData.relationship !== 'spouse'
+                hideIf: (pageData, allData) => _.get('applicantInformation.relationship', allData) !== 'spouse'
```

Comment:
I'm not following this line: why does this function need to take both pageData and allData when it only seems to use allData?